package.json

```diff
@@ -48,7 +48,10 @@
   "scripts": {
     "clean": "tsc --build tsconfig.build.json --clean",
     "prebuild": "npm run clean",
-    "lint": "prettier --write . && eslint . --ext ts,d.ts,test.ts --fix",
+    "lint": "npm run lint:prettify && npm run lint:code && npm run lint:markdown",
```

Comment:
@dimitropoulos heads up, I went ahead and edited this line, turns out the `lint` command was failing due to missing `npm run ...` in front of each command 🤷

Reply:
uggggggggg.  haha.  thank you!  I don't seem to ever get used to the npm dx after using yarn (which doesn't require this)